src/ResourceManager/KeyVault/Commands.KeyVault/Commands/RemoveAzureKeyVaultAccessPolicy.cs

```diff
@@ -87,7 +87,7 @@ public class RemoveAzureKeyVaultAccessPolicy : KeyVaultManagementCmdletBase
             ValueFromPipelineByPropertyName = true,
             HelpMessage = "Specifies the object ID of the user or service principal in Azure Active Directory for which to remove permissions.")]
         [ValidateNotNullOrEmpty()]
-        public Guid ObjectId { get; set; }
```

Comment:
@cormacpayne The parameter type changed in order to support ADFS scenarios. Unlike in AAD, object IDs in ADFS are no longer guaranteed to be of type `Guid`. This is not a breaking change, since PowerShell will automatically convert Guids to strings. For example, the `New-AzureRmResourceGroup` cmdlet has a parameter called `Name` whose parameter type is `string`, and we can successfully pass in either a `Guid` or a `string`:
```
New-AzureRmResourceGroup -Name ([Guid]::NewGuid()) -Location "westus"
New-AzureRmResourceGroup -Name "abcdef123-4567-8901-2345-67890123abcd" -Location "westus"
```